macros/calibration/pa_calibrate.cfg

```diff
@@ -61,20 +61,20 @@ gcode:
     {% set pa_start     = params.START|default(0.02)|float %}       # starting PA value
     {% set pa_increment = params.INCREMENT|default(0.005)|float %}  # how much to increment PA per band
 
-    {% set do_raft = params.RAFT|default(1)|int %}  # whether to print a raft or not
+    {% set do_raft = params.DO_RAFT|default(1)|int %}  # whether to print a raft or not
     
-    {% set print_size      = params.SIZE|default(120)|int %}                      # width/height of the printed test
+    {% set print_size      = params.PRINT_SIZE|default(120)|int %}                # width/height of the printed test
     {% set num_bands       = params.BANDS_LIMIT|default(999)|int %}               # limit the number of bands to print. othwerwise print as many as would fit within a given size
     {% set lines_per_band  = params.LINES_PER_BAND|default(6)|int %}              # number of lines per band. half are control, another half the test
     {% set e_multiplier    = params.EXTRUSION_MULTIPLIER|default(1.25 if do_raft == 1 else 1.5)|float %},  # extrusion multiplier for line print moves
-    {% set retract_length  = params.RETRACT|default(0.6)|float %}                 # how much to retract when traveling between print moves
+    {% set retract_length  = params.RETRACT_LENGTH|default(0.6)|float %}          # how much to retract when traveling between print moves
     {% set initial_purge   = params.PURGE_MM|default(8)|int %}                    # mm of filament to purge before printing. set to 0 to disable
 
     {% set feedrate_control = params.CONTROL_SPEED|default(30)|int * 60 %}  # print feedrate of the line in the middle of the band
     {% set feedrate_outer   = params.OUTER_SPEED|default(40)|int * 60 %}    # print feedrate of the lines on the sides of the band
-    {% set feedrate_inner   = params.INNER_SPEED|default(120)|int * 60 %}   # print feedrate of the line in the middle of the band
-    {% set feedrate_travel  = params.TRAVEL_SPEED|default(200)|int * 60 %}  # travel feedrate between print moves
-    {% set feedrate_raft    = params.RAFT_SPEED|default(120)|int * 60 %}    # print feedrate for printing raft
+    {% set feedrate_inner   = params.INNER_SPEED|default(80)|int * 60 %}   # print feedrate of the line in the middle of the band
+    {% set feedrate_travel  = params.TRAVEL_SPEED|default(100)|int * 60 %}  # travel feedrate between print moves
+    {% set feedrate_raft    = params.RAFT_SPEED|default(60)|int * 60 %}    # print feedrate for printing raft

```

Comment:
I'm ok with that if you want to make it more generic with lower speed values.

But in that case, I also want to keep my own "standard" values as defaults in the `variables.cfg` file used for the `CALIBRATE` macro (as talked in the other discussion)